test/health-spec.js

```diff
@@ -0,0 +1,12 @@
+var assert = require('assert'),
+  health = require('../src/health');
+
+describe('health', function() {
```

Comment:
No reason here, usually I use `function` to avoid using parent scope in tests. I will clean this up